bot/resources/tags/codeblock.md

```diff
@@ -1,11 +1,11 @@
 ---
 embed:
-    title: "Formatting code on discord"
+    title: "Formatting code on Discord"
 ---
 Here's how to format Python code on Discord:
 
 \`\`\`py
-print('Hello world!')
+print('Code goes here on a new line')
```

Comment:
Maybe this is overthinking, but are people going to think they need to put their code inside the `print('...')`?

Reply:
I think `print("Hello world!")` was more clearly a placeholder, yeah.

Reply:
I reverted this change.